extensions/ql-vscode/src/variant-analysis/gh-api/gh-api-client.ts

```diff
@@ -26,15 +25,15 @@ export async function submitVariantAnalysis(
     repository_owners: databases.repositoryOwners,
   };
 
-  const response: OctokitResponse<VariantAnalysis> = await octokit.request(
+  const response = await octokit.request(
```

Comment:
I think we're using the incorrect method of making the request, which is why this has stopped working. `POST /repositories/:controllerRepoId/code-scanning/codeql/variant-analyses` doesn't exist in the OpenAPI types (since it's listed as `POST /repos/{owner}/{repo}/code-scanning/codeql/variant-analyses`), so we should instead be using the single argument variant of the `request` method. Something like:

```ts
  const response = await octokit.request<VariantAnalysis>({
    method: "POST",
    url: "/repositories/:controllerRepoId/code-scanning/codeql/variant-analyses",
    controllerRepoId,
    data,
  });
```

I haven't checked whether this actually works correctly, but it seems like that is cleaner and doesn't require us to cast an `any` to our custom type.